src/User.hpp

```diff
@@ -6,13 +6,15 @@
 class User
 {
 public:
+    User() = default;
```

Comment:
We don't want this constructor. What is a default user?

Reply:
User() with default it was stupid idea. Constructor removed.